berkeleyMobileiOS/Classes/Academics/AcademicsViewController/AcademicViewController.swift

```diff
@@ -71,8 +82,8 @@ class AcademicViewController: UIViewController, UITableViewDelegate, UITableView
                 }
                 
                 self.libraries = nonEmptyList as! [Library]
-                if let t = self.resourceTableView {
-                    t.reloadData()
+                if self.resourceTableView != nil {
```

Comment:
don't need the check here (`resourceTableView` is implicitly unwrapped) . if the outlet is broken, we'll just let the app crash.